datasafe-rest-impl/src/main/java/de/adorsys/datasafe/rest/impl/config/DatasafeConfig.java

```diff
@@ -250,6 +255,17 @@ AmazonS3 s3(DatasafeProperties properties) {
         return amazonS3;
     }
 
+    @Bean
+    @SneakyThrows
+    KeyStoreCreationConfig keystoreConfig(KeystoreProperties kp) {
```

Comment:
Use Strings/enums here that come from API, don't bind BouncyCastle - it is encryption-impl that binds encryption libraries